warehouse/models/staging/gtfs/gtfs_guidelines_v2/stg_gtfs_guidelines__checks_implemented.sql

```diff
@@ -0,0 +1,38 @@
+{{ config(materialized='ephemeral') }}
+WITH stg_gtfs_guidelines__checks_implemented AS (
+    SELECT {{ static_feed_downloaded_successfully() }} AS check, {{ compliance() }} AS feature
+    UNION ALL
+    SELECT {{ no_validation_errors() }}, {{ compliance() }}
```

Comment:
Shouldn't this be the only check that is listed as implemented in v2?

Reply:
Discussed offline, I am re-configuring how this works.